test/integration/admin/api_docs/account_api_docs_controller_test.rb

```diff
@@ -102,10 +91,10 @@ def test_system_name_is_not_updated
 
       logout! && login!(provider, user: member)
 
-      get admin_api_docs_services_path
-      api_docs_service_ids = assigns(:api_docs_services).map(&:id)
-      assert_includes api_docs_service_ids, api_docs_service.id
-      assert_not_includes api_docs_service_ids, forbidden_api_docs_service.id
+      # get admin_api_docs_services_path
```

Comment:
Just a question, are these commented lines supposed to go?

Reply:
Yep 👍🏻